bsp/boards/openmote-b/board.c

```diff
@@ -106,10 +125,10 @@ void board_sleep(void) {
  * The timer is divided by 32, whichs gives a 1 microsecond ticks
  */
 void board_timer_init(void) {
-    // Configure the timer
+    /* Configure the timer */
```

Comment:
comments are always `//`

Reply:
I did my best fixing those now., I see this style used many board definitions